src/services/lnd/onchain-service.ts

```diff
@@ -82,17 +110,21 @@ export const OnChainService = (
   }
 
   return {
-    getIncomingTransactions,
+    listIncomingTransactions,
+    findOnChainFee,
     createOnChainAddress,
     getOnChainFeeEstimate,
   }
 }
 
-export const extractIncomingTransactions = (
-  decoder: TxDecoder,
-  { transactions }: GetChainTransactionsResult,
-): SubmittedTransaction[] => {
-  return transactions
+export const extractIncomingTransactions = ({
+  decoder,
+  txs,
+}: {
+  decoder: TxDecoder
+  txs: GetChainTransactionsResult
```

Comment:
does the `Get` in the `GetChainTransactionsResult` make sense in that context, in term of type name?

Reply:
`GetChainTransactionsResult` is a type from `lightning` library